### ISSUE TITLE
[Issue] NPM constant errors when installing.

### ISSUE THREAD

**Issue by Toltar** [npm/npm#9194]
I keep getting constant errors installing NPM of things like unmet peer dependencies, peer dependencies missing, and more. Also anything with a cli tool doesn't install correctly. After doing the cache-clean, the removal of npm, reinstalling npm, and trying npm install npm@latest as a last resort after it still not working. I got this little error: https://gist.github.com/Toltar/5a4ffd3be8b42d388319

 I just don't understand at all. Ok I am a noob at npm, nodejs, and all this automation and building stuff; but is it really worth all the trouble of having to deal with dependencies all the time. I'm sorry I am frustrated because I use this stuff for school, freelancing on the side, and have to get personal projects to get done before the end of this year. I have little time to research about all this so if someone could either help me out or point me in the right direction that would be awesome. Also I am on Windows 10 so that could be another problem with npm since WIN10 just came out.

Let me know if there is any more info you need to help me out. I have searched like the guidelines say for duplicate issues but I don't know half of what you guys may be talking about so I apologize if I did make a duplicate.

Oh BTW I cant for some reason I cant get npm-debug.log.

I apologize about my bad spelling and grammar. I am in a rush to write this issue. Please let me know if I am doing something wrong because most likely I am. Also again I am a noob at all this stuff, so speak in plain English because I am not a humanoid but I am just human. I don't need to be wasting my mental resources figuring out what you just said or how in the hell I should do it. Sorry, but I need to know exactly what, when, where, who, and how. Because I have deadlines to meet, school work with NodeJS to do, and I don't have time to read bad documentation like the people in StackOverflow did to me. I am here for answers not BS to gain points or rep. This is the last place I have posted to because I have been on every forum, Q&A, and documentation. I just for some reason cant find why.

I thank you all in advance for your contribution.

**Comment by othiym23** [comment]
It looks like `C:\Users\mattadmin\npm-global` was created with an ACL that requires Administrator privileges to write to it (or at least was created in such a way that whatever roles you have when running npm are insufficient). Try opening an Administrator prompt and see if npm works there, and if it does, try looking at `npm-global` in Explorer and seeing if you can change the permissions to work better for you.

> ```
> ./npm: line 14: 11312 Signal 112              (core dumped) node "$basedir/node_modules/npm/bin/npm-cli.js" "$@"
> ```

This piece looks like a bug in Node itself, because npm is written in pure JavaScript and shouldn't be able to cause Node to crash hard like this. My guess is that it will go away if the permissions issues go away, but if they don't, you might want to take this up with the fine people at joyent/node / nodejs/node.

**Comment by Toltar** [comment]
Ok so basically run it on CMD or Powershell as administrator? Or could I just use git-bash like I always do as administrator.
